test/starlark_apple_static_library.bzl

```diff
@@ -71,16 +73,18 @@ starlark_apple_static_library = rule(
             allow_files = True,
         ),
         "avoid_deps": attr.label_list(
-            cfg = apple_common.multi_arch_split,
+            cfg = apple_platform_split_transition,
             default = [],
         ),
         "deps": attr.label_list(
-            cfg = apple_common.multi_arch_split,
+            cfg = apple_platform_split_transition,
         ),
         "linkopts": attr.string_list(),
-        "platform_type": attr.string(),
-        "minimum_os_version": attr.string(),
+        "platform_type": attr.string(mandatory = True),
+        "minimum_os_version": attr.string(mandatory = True),
+        "_allowlist_function_transition": attr.label(
+            default = "@bazel_tools//tools/allowlists/function_transition_allowlist",
+        ),
     },
-    cfg = apple_common.apple_crosstool_transition,
```

Comment:
not sure why but removing this seemed to be a no-op for our tests so i guess we don't have to worry about this one atm